Description: Fix spelling errors.
 * occours   -> occurs
 * mutiple   -> multiple
 * postive   -> positive
 * ommited   -> omitted
 * Quiting   -> Quitting
 * verticies -> vertices
 * exlusive  -> exclusive
Author: Bas Couwenberg <sebastic@debian.org>
Forwarded: http://gmt.soest.hawaii.edu/issues/985
Applied-Upstream: http://gmt.soest.hawaii.edu/projects/gmt/repository/revisions/17237

--- a/src/meca/pspolar.c
+++ b/src/meca/pspolar.c
@@ -478,7 +478,7 @@ int GMT_pspolar (void *V_API, int mode,
 	if (Ctrl->N.active) {
 		gmt_map_outside (GMT, Ctrl->D.lon, Ctrl->D.lat);
 		if (abs (GMT->current.map.this_x_status) > 1 || abs (GMT->current.map.this_y_status) > 1) {
-			GMT_Report (API, GMT_MSG_VERBOSE, "Point give by -D is outside map; no plotting occours.");
+			GMT_Report (API, GMT_MSG_VERBOSE, "Point give by -D is outside map; no plotting occurs.");
 			Return (GMT_NOERROR);
 		};
 	}
--- a/doc/rst/source/supplements/meca/pssac.rst
+++ b/doc/rst/source/supplements/meca/pssac.rst
@@ -125,7 +125,7 @@ Optional Arguments
 
         **r**: remove mean value
 
-    **i|q|r** can repeat mutiple times. For example, **-Frii** will convert acceleration to displacement.
+    **i|q|r** can repeat multiple times. For example, **-Frii** will convert acceleration to displacement.
     The order of **i|q|r** controls the order of the data processing.
 
 .. _-G:
@@ -134,7 +134,7 @@ Optional Arguments
     Paint positive or negative portion of traces.
     If only **-G** is used, default to fill the positive portion black.
 
-        **p**\|\ **n** controls the painting of postive **p**\ ortion or **n**\ egative portion.
+        **p**\|\ **n** controls the painting of positive **p**\ ortion or **n**\ egative portion.
         Repeat **-G** option to specify fills for positive and negative portions, respectively.
 
         **+g**\ *fill*: color to fill
@@ -180,7 +180,7 @@ Optional Arguments
 
 **-S**\ *sec_per_measure*\ [*unit*]
     Specify the time scale in *sec_per_measure* seconds per *unit* while plotting on geographic plots.
-    Use PROJ_LENGTH_UNIT if *unit* is ommited.
+    Use PROJ_LENGTH_UNIT if *unit* is omitted.
 
 .. _-T:
 
--- a/src/meca/pssac.c
+++ b/src/meca/pssac.c
@@ -170,11 +170,11 @@ GMT_LOCAL int usage (struct GMTAPI_CTRL
 	GMT_Message (API, GMT_TIME_NONE, "\t   i: integral\n");
 	GMT_Message (API, GMT_TIME_NONE, "\t   q: square\n");
 	GMT_Message (API, GMT_TIME_NONE, "\t   r: remove mean value\n");
-	GMT_Message (API, GMT_TIME_NONE, "\t   i|q|r can repeat mutiple times. -Frii will convert accerate to displacement.\n");
+	GMT_Message (API, GMT_TIME_NONE, "\t   i|q|r can repeat multiple times. -Frii will convert accerate to displacement.\n");
 	GMT_Message (API, GMT_TIME_NONE, "\t   The order of i|q|r controls the order of the data processing.\n");
-	GMT_Message (API, GMT_TIME_NONE, "\t-G Paint postive or negative portion of traces.\n");
+	GMT_Message (API, GMT_TIME_NONE, "\t-G Paint positive or negative portion of traces.\n");
 	GMT_Message (API, GMT_TIME_NONE, "\t   If only -G is used, default to fill the positive portion black.\n");
-	GMT_Message (API, GMT_TIME_NONE, "\t   [p|n] controls the painting of postive portion or negative portion. Repeat -G option to specify fills for pos/neg portion, respectively.\n");
+	GMT_Message (API, GMT_TIME_NONE, "\t   [p|n] controls the painting of positive portion or negative portion. Repeat -G option to specify fills for pos/neg portion, respectively.\n");
 	GMT_Message (API, GMT_TIME_NONE, "\t   +g<fill>: color to fill\n");
 	GMT_Message (API, GMT_TIME_NONE, "\t   +t<t0>/<t1>: paint traces between t0 and t1 only. The reference time of t0 and t1 is determined by -T option.\n");
 	GMT_Message (API, GMT_TIME_NONE, "\t   +z<zero>: define zero line. From <zero> to top is positive portion, from <zero> to bottom is negative portion.\n");
@@ -188,7 +188,7 @@ GMT_LOCAL int usage (struct GMTAPI_CTRL
 	GMT_Message (API, GMT_TIME_NONE, "\t      <alpha> > 0, multiply all traces by size*r^alpha, r is the distance range in km.\n");
 	GMT_Option (API, "O,P");
 	GMT_Message (API, GMT_TIME_NONE, "\t-Q Plot traces vertically.\n");
-	GMT_Message (API, GMT_TIME_NONE, "\t-S Specify the time scale in seconds per <unit> while plotting on geographic plots. Use PROJ_LENGTH_UNIT if <unit> is ommited.\n");
+	GMT_Message (API, GMT_TIME_NONE, "\t-S Specify the time scale in seconds per <unit> while plotting on geographic plots. Use PROJ_LENGTH_UNIT if <unit> is omitted.\n");
 	GMT_Message (API, GMT_TIME_NONE, "\t-T Time alignment. \n");
 	GMT_Message (API, GMT_TIME_NONE, "\t   +t<tmark> align all trace along time mark. Choose <tmark> from -5(b), -4(e), -3(o), -2(a), 0-9(t0-t9).\n");
 	GMT_Message (API, GMT_TIME_NONE, "\t   +r<reduce_vel> reduce velocity in km/s.\n");
--- a/src/gmt_gdalread.c
+++ b/src/gmt_gdalread.c
@@ -417,7 +417,7 @@ GMT_LOCAL int populate_metadata (struct
 							" This configuration is not supported.\n");
 			GDALClose(hDataset);
 			GDALDestroyDriverManager();
-			GMT_Report (GMT->parent, GMT_MSG_NORMAL, "Quiting with error\n");
+			GMT_Report (GMT->parent, GMT_MSG_NORMAL, "Quitting with error\n");
 			return(-1);
 		}
 
@@ -431,7 +431,7 @@ GMT_LOCAL int populate_metadata (struct
 		    || anSrcWin[1] + anSrcWin[3] > GDALGetRasterYSize(hDataset)) {
 			GMT_Report (GMT->parent, GMT_MSG_NORMAL, "Computed -srcwin falls outside raster size of %dx%d.\n",
 			            GDALGetRasterXSize(hDataset), GDALGetRasterYSize(hDataset));
-			GMT_Report (GMT->parent, GMT_MSG_NORMAL, "Quiting with error\n");
+			GMT_Report (GMT->parent, GMT_MSG_NORMAL, "Quitting with error\n");
 			return(-1);
 		}
 		Ctrl->RasterXsize = nXSize = anSrcWin[2];
--- a/src/pscontour.c
+++ b/src/pscontour.c
@@ -950,7 +950,7 @@ int GMT_pscontour (void *V_API, int mode
 				n_skipped++;
 			}
 		}
-		if (n_skipped) GMT_Report (API, GMT_MSG_VERBOSE, "Skipped %u triangles whose verticies are all outside the domain.\n", n_skipped);
+		if (n_skipped) GMT_Report (API, GMT_MSG_VERBOSE, "Skipped %u triangles whose vertices are all outside the domain.\n", n_skipped);
 	}
 	
 	if (Ctrl->C.cpt) {	/* We already read the CPT */
--- a/doc/rst/source/supplements/potential/gmtgravmag3d.rst
+++ b/doc/rst/source/supplements/potential/gmtgravmag3d.rst
@@ -56,7 +56,7 @@ Required Arguments
 
 **-F**\ *xy_file*
     Provide locations where the anomaly will be computed. Note this
-    option is mutually exlusive with **-G**.
+    option is mutually exclusive with **-G**.
 
 .. _-G:
 
